static/src/javascripts/projects/common/utils/raven.js

```diff
@@ -0,0 +1,5 @@
+/*global guardian:false */
+
+define(function () {
+    return guardian.app.raven;
```

Comment:
`window.guardian`? i kind of like the idea of being mega-explicit when we're touching the global object